webapp/components/LoginForm/LoginForm.vue

```diff
@@ -12,20 +12,21 @@
       </template>
       <h2 class="title">{{ $t('login.login') }}</h2>
       <form :disabled="pending" @submit.prevent="onSubmit">
-        <ds-input
+        <base-input
           v-model="form.email"
           :disabled="pending"
           :placeholder="$t('login.email')"
           type="email"
           name="email"
           icon="envelope"
         />
-        <ds-input
+        <base-input
           v-model="form.password"
           :disabled="pending"
           :placeholder="$t('login.password')"
           icon="lock"
-          icon-right="question-circle"
+          icon-right="arrow-down"
+          icon-right-secondary="eye"
```

Comment:
As I stated in the related issue, I think it is no good idea to overload our input component with another icon. I prefer a warning message under the password field that pops up when capslock is active. This seems to be easier to realize and we can do it directly in the LoginForm component.